src/download.jl

```diff
@@ -0,0 +1,72 @@
+using Printf
+using DataDeps
+
+# generate_dataset_url helper function
+# This function takes in a dataset code of format "GADM/<country_code>"
+# It returns the URL of the gpkg dataset of the country
+
+function generate_dataset_url(dataset_code)
+
+    # Splits GADM/IND to GADM, IND
+    dataset_provider, country_code = split(dataset_code, "/")
+    
+    # Only accepts GADM Dataset Provider
+    if dataset_provider != "GADM"
+        println("❌ Dataset Provider $dataset_provider not supported.\n")
+        println("💡 Please try \"GADM/<Code>\".")
```

Comment:
Nice use of Unicode :)

Reply:
Consider using the Logging standard library: https://docs.julialang.org/en/v1/stdlib/Logging This could be a `@warn` for example.

Reply:
![image](https://user-images.githubusercontent.com/28949397/97057102-2a1a7c80-15a8-11eb-8ab7-c49b7aee0f83.png)